desktop/assets/texts/uk-UA/settings.ftl

```diff
@@ -0,0 +1,60 @@
+network-settings = Налаштування мережі
+player-settings = Налаштування плеєра
+movie-parameters = Параметри фільму
+custom-base-url = Користувацька базова URL
+spoof-swf-url = Підробити SWF URL
+referer-url = Реферальна URL
+cookie = Cookie
+proxy = Proxy
+upgrade-http = Оновити HTTP до HTTPS
+tcp-connections = З'єднання TCP
+tcp-connections-allow = Дозволити
+tcp-connections-ask = Запитати
+tcp-connections-deny = Заперечити
+open-url-mode = Відкрити посилання
+open-url-mode-allow = Дозволити
+open-url-mode-confirm = Запитати
+open-url-mode-deny = Заперечити
+load-behavior = Поведінка при завантаженні
+load-behavior-streaming = Трансляція
+load-behavior-delayed = Затримувати
+load-behavior-blocking = Блокувати
+max-execution-duration = Максимальна тривалість виконання
+max-execution-duration-suffix = { " " } секунд
+quality = Якість
+quality-low = Низька
+quality-medium = Середня
+quality-high = Висока
+quality-best = Найкраща
+quality-high8x8 = Висока (8x8)
+quality-high8x8linear = Висока (8x8) Лінійна
+quality-high16x16 = Висока (16x16)
+quality-high16x16linear = Висока (16x16) Лінійна
+letterbox = Letterbox
+letterbox-on = Увімкнено
+letterbox-fullscreen = Тільки повноекранний режим
+letterbox-off = Вимкнено
+align = Вирівнювання сцени
+align-center = Центр
+align-left = Ліворуч
+align-right = Праворуч
+align-top = Вгорі
+align-bottom = Внизу
+align-top-left = Вгорі ліворуч
+align-bottom-left = Внизу ліворуч
+align-top-right = Вгорі праворуч
+align-bottom-right = Внизу праворуч
+align-force = Примусово
+scale-mode = Режим масштабування
+scale-mode-exactfit = Точно підходить
+scale-mode-noborder = Без кордонів
+scale-mode-noscale = Без масштабу
+scale-mode-showall = Показати все
+scale-mode-force = Примусово
+player-version = Версія плеєра
+player-runtime = Час запуску плеєра
```

Comment:
Also, this might mean something more related to "running time" - also noted this on CrowdIn.